src/schema/slack/functions/open_form.ts

```diff
@@ -27,12 +27,9 @@ export default DefineFunction(
         },
         fields: {
           type: InternalSlackTypes.form_input_object,
-          description: "Input fields to be shown on the form",
```

Comment:
Huh, should these be removed?

Reply:
This will eventually be added back when the script is ran again, but these are generated changes

Reply:
Do we know why or how these made it into the PR? Was it me? 😅

Reply:
Changing the contents of the `function.json` could easily have caused a change in the generated files

Reply:
But someone would have had to run the generation script to change these files. IMO we shouldn't commit these changes

Reply:
It was hard for me to figure out where this blame came from haha. Should I just put these descriptions back ?

Reply:
Yes please, none of our built-ins should have changes in this PR

Reply:
Done in bdbdce8